src/components/settings-table/dataset-table-row.js

```diff
@@ -25,7 +25,8 @@ const DataSetTableRow = ({
                     {dataRow.option} ({periodType})
                 </p>
                 <em className={tableTitleStyles.attributeLabel}>
-                    {i18n.t('Default:')} {defaultValue}
+                    {i18n.t('Default')} :{' '}
+                    {i18n.t('{{defaultValue}}', { defaultValue })}
```

Comment:
After seeing what ends up in en.pot, I think this way of doing it will be really hard for the translators to figure out. Do you get the error if you escape the separator?  e.g. `\:`. If not, maybe a better way of doing this would be:
`i18n.t('Default{{separator}} {{defaultValue}}', {separator: ":", defaultValue})`

Reply:
I get the error if I use the separator. I'll try it using the other option